From b570dc1a2a0d30a462b9da51ffc221d911252e2d Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Tue, 24 Jan 2017 13:48:10 -0600
Subject: [PATCH 188/205] ARM: dts: AM33xx: Add PRU mboxes to mailbox node

Add the reference sub-mailbox nodes to communicate between MPU
and PRU0 (Mailbox User Id 1) and PRU1 (Mailbox User Id 2) remote
processor cores within the PRU-ICSS subsystem. The following are
the assignments of mailbox fifos between MPU and the remote PRU
cores:

		MPU Tx/PRU Rx	MPU Rx/PRU Tx
	PRU0	2		3
	PRU1	4		5

All these mailboxes provide an alternate communication path between
the MPU and PRU. The current default approach is to use the PRU
System Events.

Signed-off-by: Suman Anna <s-anna@ti.com>
(cherry picked from commit 98898951947bbba0f2126753517e8c8579c0d123)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am33xx.dtsi |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index f1a3cce..8022d2b 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -529,6 +529,14 @@
 				ti,mbox-tx = <0 0 0>;
 				ti,mbox-rx = <0 0 3>;
 			};
+			mbox_pru0: mbox_pru0 {
+				ti,mbox-tx = <2 0 0>;
+				ti,mbox-rx = <3 0 0>;
+			};
+			mbox_pru1: mbox_pru1 {
+				ti,mbox-tx = <4 0 0>;
+				ti,mbox-rx = <5 0 0>;
+			};
 		};
 
 		timer1: timer@44e31000 {
-- 
1.7.9.5

